pkg/api/v1/defaults.go

```diff
@@ -92,6 +92,15 @@ func addDefaultingFuncs() {
 			// If limits are specified, but requests are not, default requests to limits
 			// This is done here rather than a more specific defaulting pass on ResourceRequirements
 			// because we only want this defaulting semantic to take place on a Pod and not a PodTemplate
+			_, found := obj.Annotations["kubernetes.io/config.mirror"]
```

Comment:
IRL discussion w/ @yujuhong & @dchen1107 -- we're going to check for a hash to tell whether the kubelet is 1.0 or 1.1.